static/app/components/events/featureFlags/utils.tsx

```diff
@@ -115,31 +115,14 @@ export const sortedFlags = ({
 
 export enum ProviderOptions {
   LAUNCHDARKLY = 'LaunchDarkly',
-  OPENFEATURE = 'OpenFeature',
 }
 
-type Labels = {
-  pythonIntegration: string; // what's in the integrations array
-  pythonModule: string; // what's imported from sentry_sdk.integrations
-};
+export enum IntegrationOptions {
+  LAUNCHDARKLY = 'LaunchDarkly',
+  OPENFEATURE = 'OpenFeature',
+}
 
-// to organize this better, we could do something like
-// [ProviderOptions.LAUNCHDARKLY]: {
-//    python: {
-//        module: 'launchdarkly',
-//        integration 'LaunchDarklyIntegration',
-//    },
-//    javascript: {
-//        ...
-//    }
-// }
-export const PROVIDER_OPTION_TO_LABELS: Record<ProviderOptions, Labels> = {
-  [ProviderOptions.LAUNCHDARKLY]: {
-    pythonModule: 'launchdarkly',
-    pythonIntegration: 'LaunchDarklyIntegration',
-  },
-  [ProviderOptions.OPENFEATURE]: {
-    pythonModule: 'OpenFeature',
-    pythonIntegration: 'OpenFeatureIntegration',
-  },
+export const PROVIDER_OPTION_TO_URLS: Record<ProviderOptions, string> = {
+  [ProviderOptions.LAUNCHDARKLY]:
+    'https://app.launchdarkly.com/settings/integrations/webhooks/new?q=Webhooks',
```

Comment:
what happened to open feature?

Reply:
so i had to separate out the two terms: **integration** and **provider**. in my onboarding code, **integration** refers to either open feature or whatever sdk is selected in the second dropdown. so if OF is selected, then the integration is OF. the **provider** would be launchdarkly (the second dropdown). 

<img width="345" alt="SCR-20241121-kkpf" src="https://github.com/user-attachments/assets/d82ba3e9-6589-4c31-9165-0e15ba4ebbfb">

if the second radio button is clicked, then the **integration** and **provider** are both LD. 

when we add more options to the OF second dropdown (say "custom"), then we'll add to the provider list. but those aren't integrations ("custom" would not be an integration)